examples/znap-hats/app/web/app/actions.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "znap-action"
```

Comment:
just an example.. ill replace this with real data once we have it.

Reply:
use this for now:

```json
{
  "rules": [
    {
      "pathPattern": "/**",
      "apiPath": "/api/**"
    }
  ]
}
```